src/styles/header.css

```diff
@@ -6,12 +6,12 @@
     z-index: 3;
     min-height: 7vh;
     height: 8vh;
-    background-color: #d3d3d3;
+    background-color: #f7f7f8;
     padding-top: 3.5vh;
     transition: transform 0.5s ease-in;
     -webkit-transition: transform 0.5s ease-in;
-    -webkit-box-shadow: 2px 2px 2px 0px rgba(50, 50, 50, 0.75);
-    box-shadow: 2px 2px 2px 0px rgba(50, 50, 50, 0.75);
+    -webkit-box-shadow: 1px 1px 1px 0px #b2b2b2;
```

Comment:
because the image in zeplin wasn't available.

![alt text](https://lh5.googleusercontent.com/3-qukZ89sXRT-lCw46YQexPMvk1f882_aUlzSDBKB1RMVhMyHr511Eo39MKD35lbOjgVpULz5NTkwZA=w1287-h539)